commit 3fda2738b0c78df0454e72691a43b459c2e57c21
Author: James Antill <james@and.org>
Date:   Tue Mar 11 15:09:18 2014 -0400

    Change man page text for RHEL-7 group_command=object feedback.

diff --git a/docs/yum.8 b/docs/yum.8
index 6794581..1ab8534 100644
--- a/docs/yum.8
+++ b/docs/yum.8
@@ -250,20 +250,34 @@ Is used to download and make usable all the metadata for the currently enabled
 sure the repos. are current (much like "yum clean expire-cache").
 .IP 
 .IP "\fBgroups\fP"
-A command, new in 3.4.2, that collects all the subcommands that act on groups together.
-
-"\fBgroup install\fP" is used to install all of the individual packages in a group, of the specified
-types (this works as if you'd taken each of those package names and put them on
-the command line for a "yum install" command).
+A command, new in 3.4.2, that collects all the subcommands that act on groups
+together. Note that recent yum using distributions (Fedora-19+, RHEL-7+) have
+configured group_command=objects which changes how group commands act in some
+important ways.
+
+"\fBgroup install\fP" is used to install all of the individual packages in a
+group, of the specified types (this works as if you'd taken each of those
+package names and put them on the command line for a "yum install" command).
  The group_package_types configuration option specifies which types will
 be installed.
-
-"\fBgroup update\fP" is just an alias for groupinstall, which will do the right thing because
-"yum install X" and "yum update X" do the same thing, when X is already
-installed.
-
-"\fBgroup list\fP" is used to list the available groups from all \fByum\fP repos. Groups are marked
-as "installed" if all mandatory packages are installed, or if a group doesn't
+ If you wish to "reinstall" a group so that you get a package that is currently
+blacklisted the easiest way to do that currently is to install the package
+manually and then run "groups mark packages-sync mygroup mypackagename" (or
+use yumdb to set the group_member of the package(s)).
+
+"\fBgroup update\fP" is just an alias for group install, when using
+group_command=compat. This will install packages in the group not already
+installed and upgrade existing packages. With group_command=simple it will just
+upgrade already installed packages. With group_command=objects it will try to
+upgrade the group object, installing any available packages not blacklisted
+(marked '-' in group info) and will upgrade the installed packages.
+
+"\fBgroup list\fP" is used to list the available groups from all \fByum\fP
+repos. When group_command=objects the group is installed if the user
+explicitly installed it (or used the group mark* commands to mark it installed).
+It does not need to have any packages installed.
+When not using group_command=objects groups are shown as "installed" if all
+mandatory packages are installed, or if a group doesn't
 have any mandatory packages then it is installed if any of the optional or
 default package are installed (when not in group_command=objects mode).
 You can pass optional arguments to the list/summary commands: installed,
@@ -300,6 +314,9 @@ meaning of these markers is:
 .br
 "=" = Package is installed, and was installed via the group.
 
+you can move an installed package into an installed group using either
+"group mark package-sync/package-sync-forced" or "yumdb set group_member".
+
 "\fBgroup summary\fP" is used to give a quick summary of how many groups
 are installed and available.
 
commit e15943868e2a05e4304247f1e19d2520701e9cca
Author: James Antill <james@and.org>
Date:   Tue Mar 25 00:12:26 2014 -0400

    Documentation tweak for group info and blacklisted packages.

diff --git a/docs/yum.8 b/docs/yum.8
index 1ab8534..3f028f8 100644
--- a/docs/yum.8
+++ b/docs/yum.8
@@ -306,7 +306,7 @@ to each package saying how that package relates to the group object. The
 meaning of these markers is:
 
 .br
-"-" = Package isn't installed, and won't be installed as part of the group (Eg. group install foo -pkgA … this will have pkgA marked as '-')
+"-" = Package isn't installed, and won't be installed as part of the group (Eg.  "yum group install foo -pkgA" or "yum group install foo; yum remove pkgA" … this will have pkgA marked as '-')
 .br
 "+" = Package isn't installed, but will be the next time you run "yum upgrade" or "yum group upgrade foo"
 .br
